tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -1276,7 +1289,7 @@ Status TRTEngineOp::AllocateCalibrationResources(
         partial_shapes, &cache_res->GetLogger(), cache_res->allocator_.get(),
         cres->calibrator_.get(), &cres->engine_, /*use_calibration=*/true,
         this->use_implicit_batch_, /*convert_successfully=*/nullptr,
-        /*profiles=*/nullptr, name());
+        /*profiles=*/nullptr, name(), nullptr);  // TODO(spanev): cluster + INT8 = ?
```

Comment:
Would you please create a github ticket and describe the TODO there, then use the ticket number here as TODO(path to github ticket)?